packages/http-signature-utils/src/utils/headers.ts

```diff
@@ -0,0 +1,50 @@
+import { createContentDigestHeader } from 'httpbis-digest-headers'
+import {
+  createSignatureHeaders,
+  SignatureHeaders,
+  SignOptions
+} from './signatures'
+
+interface ContentHeaders {
+  'Content-Digest': string
+  'Content-Length': string
+  'Content-Type': string
+}
+
+export interface Headers extends SignatureHeaders, Partial<ContentHeaders> {}
+
+const createContentHeaders = (body: string): ContentHeaders => {
+  return {
+    'Content-Digest': createContentDigestHeader(
+      JSON.stringify(JSON.parse(body)),
+      ['sha-512']
+    ),
+    'Content-Length': Buffer.from(body as string, 'utf-8').length.toString(),
+    'Content-Type': 'application/json'
+  }
+}
+
+export const createHeaders = async ({
```

Comment:
Add tests

Reply:
I don't think that is necessary since all it does is call `createContentDigestHeader`, which is imported from `httpbis-digest-headers`, and `createSignatureHeaders`, which is already tested. What do you think?